trader.py

```diff
@@ -43,85 +29,29 @@ class Trader:
-        """
-        Only method required. It takes all buy and sell orders for all symbols as an input,
-        and outputs a list of orders to be sent
-        """
-        # Initialize the method output dict as an empty dict
-        result = {}
 
-        # Iterate over all the keys (the available products) contained in the order depths
-        for product in state.listings.keys():
-            if product != "PEARLS":
-                continue
-
-            # Retrieve the Order Depth containing all the market BUY and SELL orders for PEARLS
-            order_depth: OrderDepth = state.order_depths[product]
-
-            # Initialize the list of Orders to be sent as an empty list
-            orders: list[Order] = []
-
-            # Define a fair value.
-            if product in state.market_trades:
-                self.trade_prices[product].extend(state.market_trades[product])
-            fair_value = self.calculate_fair_value(product)
-            if fair_value is None:
-                continue
-
-            # If there are any SELL orders
-            if len(order_depth.sell_orders) > 0:
-
-                # Sort all the available sell orders by their price,
-                # and select only the sell order with the lowest price
-                best_ask = min(order_depth.sell_orders.keys())
-                best_ask_volume = order_depth.sell_orders[best_ask]
-
-                # Check if the lowest ask (sell order) is lower than the above defined fair value
-                if best_ask < fair_value:
-
-                    # In case the lowest ask is lower than our fair value,
-                    # This presents an opportunity for us to buy cheaply
-                    # The code below therefore sends a BUY order at the price level of the ask,
-                    # with the same quantity
-                    # We expect this order to trade with the sell order
-                    print("BUY", str(-best_ask_volume) + "x", best_ask)
-                    orders.append(
-                        Order(product, best_ask, -best_ask_volume))
-
-            # The below code block is similar to the one above,
-            # the difference is that it find the highest bid (buy order)
-            # If the price of the order is higher than the fair value
-            # This is an opportunity to sell at a premium
-            if len(order_depth.buy_orders) != 0:
-                best_bid = max(order_depth.buy_orders.keys())
-                best_bid_volume = order_depth.buy_orders[best_bid]
-                if best_bid > fair_value:
-                    print("SELL", str(best_bid_volume) + "x", best_bid)
-                    orders.append(
-                        Order(product, best_bid, -best_bid_volume))
-
-            # Add all the above the orders to the result dict
-            result[product] = orders
+        result = {}
+        result["PEARLS"] = self.trade_pearls(state.order_depths["PEARLS"])
```

Comment:
@jason2594746793 you can create a method called `trade_coconuts_and_pina_coladas` like this